challenge.rb

```diff
@@ -67,6 +90,20 @@ def ac_monitor current_temp, ac_working, desired_temp
 # and print out the result of `fizz_buzz_calculator` for that number.
 # This question will be graded independently of question four.
 
+fizz_buzz(min, max)
+  while min <= max
+      counter += 1
+      if counter%15 == 0
+        puts "fizzbuzz"
+      elsif counter%3 == 0
+        puts "fizz"
+      elsif counter%5 == 0
+        puts "fizzbuzz"
+      else
+          puts counter
+      end
+  end
+end
```

Comment:
Unfortunately, this solution won't work unless you start off by setting counter to min. Also, it doesn't call another method, `fizz_buzz_calculator`, to figure out the right response.